.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files

```

Comment:
Cały katalog `.idea` powinien znaleźć się w pliku `.gitignore`.

Jest to katalog, którego PyCharm używa do przechowywania róznych metadanych o projekcie. Jako że te metadane będą się rózniły w zależności od środowiska, nie nalezy go wersjonować.